src/index.scss

```diff
@@ -1,9 +1,10 @@
+@import 'carbon-components/scss/globals/scss/styles.scss';
+
 $feature-flags: (
   ui-shell: true,
+  grid-columns-16: true,
 );
 
-@import 'carbon-components/scss/globals/scss/styles.scss';
-
 // Remove overrides once Carbon bugs are fixed upstream.
```

Comment:
The feature flags mapping should come before the styles import. Your landing page background images is not rendering properly,

Reply:
I have move the feature flags at top before the carbon styles.